stablehlo/reference/Ops.cpp

```diff
@@ -149,109 +166,114 @@ SmallVector<Tensor> eval(
 
   Scope scope(parent);
   scope.add(block.getArguments(), args);
-
   for (Operation &op : block) {
     if (auto absOp = dyn_cast<AbsOp>(op)) {
-      auto operand = scope.find(absOp.getOperand());
+      auto operand = scope.find(absOp.getOperand()).getTensor();
```

Comment:
Having seen how pervasive getTensor/getTensors are in real life, I think that we should have `find`, `findTensor`, `findTensors`, `findToken` and `findTokens` on Scope. This would considerably reduce the visual noise. You were right to propose this in the first place.